sormas-api/src/main/java/de/symeda/sormas/api/utils/jurisdiction/CaseJurisdictionHelper.java

```diff
@@ -17,39 +17,40 @@
  *******************************************************************************/
 package de.symeda.sormas.api.utils.jurisdiction;
 
-import java.util.Collections;
+import java.util.Set;
 
 import de.symeda.sormas.api.caze.CaseJurisdictionDto;
+import de.symeda.sormas.api.user.JurisdictionLevel;
 import de.symeda.sormas.api.user.UserRole;
 import de.symeda.sormas.api.utils.DataHelper;
 
 public class CaseJurisdictionHelper {
 
-	public static Boolean isInJurisdiction(RoleCheck roleCheck, UserJurisdiction userJurisdiction, CaseJurisdictionDto caseJurisdictionDto) {
+	public static Boolean isInJurisdiction(Set<UserRole> userRoles, UserJurisdiction userJurisdiction, CaseJurisdictionDto caseJurisdictionDto) {
```

Comment:
Instead of passing the user roles directly pass the JurisdictionLevel and use a switch statement.

Reply:
ok